applications/frontend/src/components/header/Header.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+
+import { makeStyles } from '@material-ui/core/styles';
+import { Toolbar, Box, Typography, Button, IconButton } from '@material-ui/core';
+import PersonIcon from '@material-ui/icons/Person';
+import SearchIcon from '@material-ui/icons/Search';
+
+import { toggleDrawer } from '../../store/reducers/drawer';
```

Comment:
Don't expect to see a dependency to a reducer from inside a component

Reply:
:-) forgot to remove this one... it is old and not used anymore